plugins/sqlfluff-templater-dbt/test/fixtures/dbt/dbt_project/models/my_new_project/L037_test.sql

```diff
@@ -0,0 +1,8 @@
+select
+    birth_date,
+    name
+from cows
+order by
```

Comment:
I think the original issue you mentioned involved a `TemplateLoop`. Should the SQL file have a `{% for %}` loop in order to reproduce the issue?

Reply:
Interestingly enough the original issue I found this in had no templating. Not sure why that was in the parse results - maybe a separate issue?

Reply:
@alanmcruickshank: Any ideas on this?

To me, as long as the PR fixes _an_ issue that is covered by newly added tests, that's progress, and if something arises later, we can address it at that time.

Reply:
agreed